pkg/mdformatter/linktransformer/link.go

```diff
@@ -172,9 +186,59 @@ func NewValidator(logger log.Logger, except *regexp.Regexp, anchorDir string) (m
 	return v, nil
 }
 
+type GitHubResponse struct {
+	Number int `json:"number"`
+}
+
+func getGitHubRegex(reponame string) (*regexp.Regexp, int, error) {
```

Comment:
Let's make it a separate validator, totally separate file etc as this is complex. For readability it's best to isolate complexities from each other and abstract them away (:

Reply:
Sure!